docs/arlas-api-exploration.md

```diff
@@ -14,7 +14,7 @@ The table below lists the URL endpoints and their optional "parts". A part is co
 | /arlas/explore/`{collection}`/**_range**?`field` & `filter` & `form` | Calculates the min and max values of a field in the collection, given the filters |
 | /arlas/explore/`{collection}`/**_search**?`filter` & `form` & `projection` & `page` | Search and return the elements found in the collection, given the filters |
 | /arlas/explore/`{collection}`/**_geosearch**?`filter` & `form` & `projection` & `page` | Search and return the elements found in the collection as features, given the filters |
-| /arlas/explore/`{collection}`/**_geosearch**/`{z}`/`{x}`/`{y}`?`filter` & `form` & `projection` & `page` | Search and return the elements found in the collection and localized in the given tile(x,y,z) as features, given the filters |
+| /arlas/explore/`{collection}`/**_tile**/`{z}`/`{x}`/`{y}`?`filter` & `form` & `projection` & `page` | Search and return the elements found in the collection and localized in the given tile(x,y,z) as features, given the filters |
```

Comment:
There are both `_geosearch`, and `_geosearch/{z}/{x}/{y}`
`_tile`exists but it's another endpoint : `{collection}/_tile/{z}/{x}/{y}.png`

Reply:
Added `| /arlas/explore/`{collection}`/**_tile**/`{z}`/`{x}`/`{y}`.png?`filter` & `form` & `projection` & `page` | Return the PNG of the elements found in the collection and localized in the given tile(x,y,z) as features, given the filters |
`